# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../silo/v7.patch.sparc
# Copyright (C) 2017 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- silo.git/common/Makefile.vanilla	2017-12-23 10:34:40.583625829 +0000
+++ silo.git/common/Makefile	2017-12-23 10:34:59.651625131 +0000
@@ -8,7 +8,7 @@
 	$(CC) $(CFLAGS) -c $*.c
 
 .S.o:
-	$(CC) $(CFLAGS) -c $*.S
+	$(CC) $(CFLAGS) -mcpu=v8 -c $*.S
 
 OBJS = sdiv.o rem.o udiv.o urem.o jmp.o printf.o console.o prom.o tree.o stringops2.o\
 	stringops1.o ffs.o divdi3.o udivdi3.o
--- silo.git/second/Makefile.vanilla	2017-12-23 10:35:36.947623766 +0000
+++ silo.git/second/Makefile	2017-12-23 10:35:55.527623086 +0000
@@ -30,7 +30,7 @@
 	$(CC) $(CFLAGS) -c $*.c -o $@
 
 .S.o:
-	$(CC) $(CFLAGS) -c $*.S -o $@
+	$(CC) $(CFLAGS) -mcpu=v8 -c $*.S -o $@
 
 # The ordering here is very significant. Please add new object files to OBJS5 only.
 OBJS1 = crt0.o
--- silo.git/Makefile.vanilla	2017-12-23 10:36:45.879621242 +0000
+++ silo.git/Makefile	2017-12-23 10:38:25.547617594 +0000
@@ -3,7 +3,7 @@
 
 # These only get built on Linux
 ifeq ($(OPSYS),Linux)
-  SUBDIRS  = common first second first-isofs tilo
+  SUBDIRS  = common first second first-isofs # tilo
   MANPAGES = maketilo.1 tilo.1
 endif
 
@@ -24,8 +24,8 @@
 ifeq ($(OPSYS),Linux)
 	install -d -m755 $(DESTDIR)/boot $(DESTDIR)/usr/bin
 	install -m644 first/*.b second/*.b first-isofs/*.b $(DESTDIR)/boot/
-	install -m755 tilo/maketilo $(DESTDIR)/usr/bin/
-	install -m755 tilo/tilo.sh $(DESTDIR)/usr/bin/tilo
+	#install -m755 tilo/maketilo $(DESTDIR)/usr/bin/
+	#install -m755 tilo/tilo.sh $(DESTDIR)/usr/bin/tilo
 endif
 	for manpage in $(MANPAGES); do \
 		sect=`echo $$manpage | sed 's/.*\([1-8]\)$$/\1/'`; \
